include/rocksdb/utilities/optimistic_transaction_db.h

```diff
@@ -31,6 +31,28 @@ struct OptimisticTransactionOptions {
   const Comparator* cmp = BytewiseComparator();
 };
 
+enum class OccValidationPolicy {
+  // Validate serially at commit stage, AFTER entering the write-group.
+  // Isolation validation is processed single-threaded(since in the
+  // write-group).
+  // May suffer from high mutex contention, as per this link:
+  // https://github.com/facebook/rocksdb/issues/4402
+  VALIDATE_SERIAL = 0,
```

Comment:
Our naming convenstion is `kValidateSerial`.

Reply:
> Thank you for your contribution! I have some final comments. Most of them are about code styles. We are almost ready to go. Please mention the improvement in HISTORY.md.

thanks for your patience. I've resolved this. you can find it in HISTORY.md in the diff.

Reply:
> Our naming convenstion is `kValidateSerial`.

resolved